src/main/java/de/teamlapen/vampirism/entity/BlindingBatEntity.java

```diff
@@ -49,4 +51,17 @@ public void tick() {
     public void restrictLiveSpan() {
         this.restrictLiveSpan = true;
     }
+
+    public static boolean spawnPredicate(EntityType<? extends BlindingBatEntity> p_223369_0_, IWorld p_223369_1_, SpawnReason p_223369_2_, BlockPos p_223369_3_, Random p_223369_4_) {
```

Comment:
Usually nice to refractor the params to something more readable if copying decompiled code